src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -13,6 +13,9 @@
         </a></li>
       {% endif %}
 
+      <li class="show-on-desktop"><a href="https://extensionworkshop.com/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=devhub-menu-link">
+          {{ _('Extension Workshop') }}
```

Comment:
@eviljeff I made the changes, but as per bug description (maybe it needs to be updated), we might need to update the logged in view. If not lets 🚢 it.

Thanks